src/components/VideoPlayer/index.js

```diff
@@ -93,6 +93,16 @@ export default class VideoPlayer extends PureComponent {
     }
   }
 
+  componentWillUnmount () {
+    // clean listeners
+    this.video.on('play', () => {})
```

Comment:
Should this use `this.video.off('play')` instead?

Reply:
yeah, good catch. It was fixed!